pyperformance/data-files/benchmarks/bm_coroutines/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "pyperformance_bm_coroutines"
+requires-python = ">=3.8"
+dependencies = ["pyperf"]
+urls = {repository = "https://github.com/python/pyperformance"}
+dynamic = ["version"]
+
+[tool.pyperformance]
+name = "coroutines"
```

Comment:
(I guess eventually the microbenchmarks will need a tag they share, but that's not for this PR.)